APIs/openEO/job_config.qmd

```diff
@@ -43,6 +43,8 @@ This is a short overview of the various options:
 - `logging-threshold`: the threshold for logging, set to 'info' by default, can be set to 'debug' to generate much more logging
 - `udf-dependency-archives`: an array of urls pointing to zip files with extra dependencies, see below
 
+If you're interested in delving further into the terminology associated with Kubernetes and how they are used, please have a look at [this page](https://kubernetes.io/docs/concepts/configuration/manage-resources-containers/){target="_blank"}, which provides a detailed walkthrough of the general idea on resource allocation.
```

Comment:
I'm not sure that kubernetes doc page is very actionable for a regular user. It's not trivial to bridge the gap between openeo job options and all the kubernetes details explained there

Reply:
If so, sure, I can avoid doing this. I was just curious if it is relevant for users like me with limited understanding in setting up the jobs.